### PR TITLE
feat(cli): Note that you can use sh

### PR DESCRIPTION
**Pull request by evanpurkhiser**
image

### CODE REVIEW

**Review by mitsuhiko — CHANGES_REQUESTED** [review]
Just change it to `sh` as a default. No need to mention bash.

File: src/docs/product/cli/installation.mdx [hunk 1]
```diff
@@ -29,6 +29,12 @@ curl -sL https://sentry.io/get-cli/ | SENTRY_CLI_VERSION="{{ apps.version('sentr
 
 This will automatically download the correct version of `sentry-cli` for your operating system and install it. If necessary, it will prompt for your admin password for `sudo`. For a different installation location or for systems without `sudo` (like Windows), you can `export INSTALL_DIR=/custom/installation/path` before running this command.
 
+<Note>
+
+You may also wish to use `sh` over `bash` if you are in a constrained environment.
+
+</Note>
```
Comment (lizokm) [inline src/docs/product/cli/installation.mdx:36]:
```suggestion
<Note>

If you're in a constrained environment, you may wish to use `sh` over `bash`.

</Note>
```

**Review by lizokm — APPROVED** [review]
LGTM

**Comment by evanpurkhiser** [comment]
Closing in favor of https://github.com/getsentry/sentry-docs/pull/6067

**Pull request closed without merging**